package.json

```diff
@@ -17,14 +17,17 @@
     "module_name": "index",
     "host": "https://github.com/thefrontside/ctrlc-windows/releases/download/",
     "remote_path": "v{version}",
-    "package_name": "{node_abi}-{platform}-{arch}.tar.gz",
-    "module_path": "./dist"
+    "package_name": "{platform}-{arch}-napi-v{napi_build_version}.tar.gz",
+    "module_path": "./dist/napi-{napi_build_version}",
```

Comment:
The way `node-pre-gyp` works with `napi`, it requires a module_path with that `{napi_build_version}`.